src/components/Search/CompareWithBase.tsx

```diff
@@ -14,8 +14,8 @@ import type { Changeset, Repository } from '../../types/state';
 import { Framework } from '../../types/types';
 import CancelAndCompareButtons from './CancelAndCompareButtons';
 import EditButton from './EditButton';
-import FrameworkDropdown from './FrameworkDropdown';
 import SearchComponent from './SearchComponent';
+import SearchFrameworkComponent from './SearchFrameworkComp';
```

Comment:
nit: use `SearchFrameworkDropdown`, so that it's clearer what this is when looking at the code here.